### PR TITLE
PO.DAAC Portal

### PR DESCRIPTION
**Pull request by lewismc**
This PR is a re submission of the original PR #703 
As you can see, I've also alphabetically ordered entries in ```config/portals.yml```
Can someone post the URL-path to portals?
I think that the addition of a portal should be documented somewhere so I'll probably start work on that. Thanks in advance for any review.

### CODE REVIEW

**Comment by littleidiot** [comment]
https://search.earthdata.nasa.gov/portal/<portal name>/search

For example: https://search.earthdata.nasa.gov/portal/cwic/search

**Comment by lewismc** [comment]
Ta @littleidiot

**Comment by lewismc** [comment]
@littleidiot I started creating documentation for this item at https://github.com/nasa/earthdata-search/wiki/Adding-a-Portal-to-EDSC, do you want me to finish it or do you have existing documentation elsewhere?

**Comment by littleidiot** [comment]
This: https://wiki.earthdata.nasa.gov/display/EDSC/Portal+Creation+Guide
And iirc I had some comments on jquery + knockout problems.

File: config/portals.yml [hunk 1]
```diff
@@ -50,19 +65,19 @@ portals: &portals
...
-        - edsc.portal.cwic.collections
+      data_center:
+        - PODAAC
     logos:
-      - id: ceos-logo
-        image: /images/portals/cwic/ceos-logo.png
-        link: http://ceos.org/
-        title: CEOS Home
+      - id: podaac-logo
+        image: /images/portals/podaac/podaac-logo-mono.png
```
Comment (macrouch) [inline config/portals.yml]: This is the wrong filename for the logo you added. `podaac-logo` vs `podaac_logo`

File: config/portals.yml [hunk 2]
```diff
@@ -50,19 +65,19 @@ portals: &portals
...
-  cwic:
-    org: CWIC
+  podaac:
+    org: PO.DAAC
     title: Search
     params:
-      tag-key:
-        - edsc.portal.cwic.collections
+      data_center:
+        - PODAAC
```
Comment (macrouch) [inline config/portals.yml]: I'm not seeing any collection results from this parameter, did you mean `PO.DAAC`?

**Comment by lewismc** [comment]
@littleidiot ... any motivation to make this document public? It would be encouraging to see it referenced from the README or something...

**Comment by littleidiot** [comment]
I have no objection to that. Check with @mreese84 though

**Comment by abbottry** [comment]
@lewismc is anything using `podaac_logo-color.png`? I don't see it mentioned and would rather not have extra binary objects sitting in the repository.

**Comment by lewismc** [comment]
@abbottry just removed it. Good call. Thanks for review.

**Comment by lewismc** [comment]
@abbottry thanks, can you detail when and where we will see the portal? I would like to sit down with my team here and begin designing what we want it to do and look like. Thanks

**Comment by lewismc** [comment]
Excellent, thank you @mreese84

**Comment by termux-io** [comment]
[import MapView from 'react-native-maps';](https://uline.mx therealcurlsport)

**Pull request merged**